cards/cards.css

```diff
@@ -0,0 +1,160 @@
+:root{
+    --bkg-clr:linear-gradient(45deg,aqua  0%, rgba(182, 241, 252, 1) 50%, rgba(252, 174, 241, 1) 70%);
+    --sec-text-clr:gray;
+    --pri-text-clr:black;
+    --danger-clr:red;
+}
+
+img {
+    max-inline-size: 100%;
+    block-size: auto;
+    border-radius: 1rem 1rem 0rem 0rem;
+  }
+  
+  img:hover {
+    cursor: pointer;
+  }
+  
+  .card {
+    background: var(--bkg-clr);
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 350px;
+    height: min-content;
+    border-radius: 1rem;
+  }
+  
+  .card:hover {
+    transition: 0.5sec;
+    box-shadow: 2px 2px 2px 2px;
+  }
+  
+  .text {
```

Comment:
Replace .text with .text-container because it contains multiple text blocks.

Reply:
replaced